### PR TITLE
fix typos in README.md

### PR DESCRIPTION
**Pull request by rahul0x00**
I found a few typos and improvements in README.md

1. In the "Download" section, consider capitalizing the first letter for consistency: "Check the Downloads page on podman-desktop.io/downloads."

2. In the "Contributing" section, consider making the section titles more consistent by capitalizing the first letter: ":bug: File Bugs or Feature Requests on GitHub" and ":checkered_flag: Read Our Contributing Guide".

3. In the "Communication" section, you might want to clarify the phrasing: "All channels are bridged. Chat on either Discord, IRC, or Matrix, and it will appear on all three!"

### CODE REVIEW

**Comment by rahul0x00** [comment]
Other Suggestion: 
"Join our early adopter program! (optional)" section, we can also make the phrase "optional" more visually distinct, like this: "This is entirely optional and does not impact any Podman Desktop features."

**Review by jeffmaury — COMMENTED** [review]
Not sure about the second section (File Bugs...)

**Review by themr0c — CHANGES_REQUESTED** [review]
We write all titles and headings, including the titles of product documentation guides and Knowledgebase articles, in sentence-style capitalization. Do not use headline-style capitalization.

See: https://redhat-documentation.github.io/supplementary-style-guide/#titles-and-headings

**Comment by rahul0x00** [comment]
> Not sure about the second section (File Bugs...)

In sentence-style capitalization, it's better to capitalize the first word of the heading and any proper nouns, so "File Bugs" is the appropriate choice.

File: README.md [hunk 1]
```diff
@@ -81,9 +81,9 @@ Check out all our [future features!](https://github.com/containers/podman-deskto
 
 Interested in fixing issues or contributing to Podman Desktop?
 
-- :bug: [File bugs or feature requests on GitHub](https://github.com/containers/podman-desktop/issues/new/choose)
+- :bug: [File Bugs or feature requests on GitHub](https://github.com/containers/podman-desktop/issues/new/choose)
```
Comment (themr0c) [inline README.md]: This sentence is not a title.

File: README.md [hunk 2]
```diff
@@ -81,9 +81,9 @@ Check out all our [future features!](https://github.com/containers/podman-deskto
...
 - :checkered_flag: [Read our contributing guide](./CONTRIBUTING.md)
-- :ok_hand: [Review or contribute a pull request](https://github.com/containers/podman-desktop/pulls)
+- :ok_hand: [Review or Contribute a Pull Request](https://github.com/containers/podman-desktop/pulls)
```
Comment (themr0c) [inline README.md]: This sentence is not a title.

File: README.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ General questions & development:
 - [#podman-desktop@libera.chat on IRC](https://libera.chat/)
 - [#podman-desktop@fedora.im on Matrix](https://chat.fedoraproject.org/#/room/#podman-desktop:fedora.im)
 
-Note: All channels are bridged. Chat on either: Discord, IRC or Matrix and it'll appear on all three!
+Note: All channels are bridged. Chat on either Discord, IRC, or Matrix, and your messages will appear on all three platforms!
```
Comment (themr0c) [inline README.md:102]: Can you update the pull request to leave only this change?

Comment (rahul0x00) [reply README.md:102]: done

**Review by themr0c — APPROVED** [review]
Nice! Thank you!

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
thanks @rahul0x00

**Pull request merged**